selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/user/TestUserImpl.java

```diff
@@ -45,17 +44,32 @@
   private final TestWorkspaceServiceClient workspaceServiceClient;
 
   /** To instantiate user with generated email and password. */
-  @Inject
+  @AssistedInject
```

Comment:
Did you test it? 
It will lead to an error of injection when we try to create random user by the command `@inject TestUser testUser;`.

Reply:
yes for usecase described [here](https://github.com/eclipse/che/pull/6456#discussion_r141295205) it works pretty fine